DigitalLearningSolutions.Data/Services/DiagnosticAssessmentDataService.cs

```diff
@@ -25,7 +25,14 @@ public DiagnosticAssessmentDataService(IDbConnection connection, ILogger<Diagnos
         }
 
         public DiagnosticAssessment? GetDiagnosticAssessment(int customisationId, int candidateId, int sectionId)
-        {
+        {
+            // NextTutorialID is the ID of the first tutorial in the section, according to Tutorials.OrderBy
+            // or null if the last in the section. Similar for NextSectionID, using SectionID and SectionNumber
+
+            // Find these by making a list of other tutorials in the course, to then find tutorials in this section,
+            // and other sections (because a section must contain at least one tutorial), using a similar approach to
+            // the one used in the TutorialContentService
+
```

Comment:
Probably worth explaining how we're using the extra views to find whether there are other sections or other items in this section 🙂

Reply:
Yep! Done in c6e65d9 🙂